lisa/tools/kernel_config.py

```diff
@@ -133,3 +133,30 @@ def module_exists(
             return True
 
         return False
+
+
+class KLDLoad(Tool):
```

Comment:
If the class is used in modprobe only, it should be in modeprobe.

Reply:
I was following the format of KLDStat as that is a similar tool. KLDStat is only used in modprobe from the references I can see in VScode. Should I move both KLDLoad and KLDStat into modprobe?

Reply:
Yes, please move them both.

Reply:
I missed one use of kldstat in kernelconfigBSD so I left it there. KLDLoad is now in modprobe.